osm_node:
  ros__parameters:
    # osm_file: "/home/acca/catkin_ws/src/localization/resource/school.osm"
    osm_file: "/home/acca/catkin_ws/src/localization/resource/kcity.osm"
    # gps_origin: [37.4961657, 126.9570535] # school
    gps_origin: [37.23963, 126.77449] # kcity
    timer_period_sec: 10.0  # zero to publish once